content/docs/operating/configuration.md

```diff
@@ -224,55 +228,41 @@ A `tls_config` allows configuring TLS connections.
 [ insecure_skip_verify: <boolean> ]
 ```
 
+### `<azure_sd_config>`
+CAUTION: Azure SD is in beta: breaking changes to configuration are still
+likely in future releases.
 
-### `<static_config>`
-
-A `static_config` allows specifying a list of targets and a common label set
-for them.  It is the canonical way to specify static targets in a scrape
-configuration.
-
-```
-# The targets specified by the static config.
-targets:
-  [ - '<host>' ]
-
-# Labels assigned to all metrics scraped from the targets.
-labels:
-  [ <labelname>: <labelvalue> ... ]
-```
-
-Where `<host>` is a valid string consisting of a hostname or IP followed by a port
-number.
-
+Azure SD configurations allow retrieving scrape targets from Azure VMs.
 
-### `<dns_sd_config>`
+The following meta labels are available on targets during relabeling:
 
-A DNS-SD configuration allows specifying a set of DNS record names which
-are periodically queried to discover a list of targets (host-port pairs). The
-DNS servers to be contacted are read from `/etc/resolv.conf`.
+* `__meta_azure_machine_id`: the machine ID
+* `__meta_azure_machine_rescource_group`: the machine's resource group
+* `__meta_azure_machine_location`: the location the machine runs in
+* `__meta_azure_machine_private_ip`: the machine's private IP
+* `__meta_azure_tag_<tagkey>`: each tag value of the machine
```

Comment:
In Azure, the used term seems to be "tag names" rather than "tag keys". (in EC2, "tag key" seems to be the more common term).